### ISSUE TITLE
No documentation how to uninstall Flux

### ISSUE THREAD

**Issue by Marx2** [fluxcd/flux#3225]
I've installed Flux in cluster using example "flux-get-started". Now I want to remove this configuration and install Flux with my own GIT repository. I've noticed that there's no description how to uninstall Flux from cluster. I don't see also uninstall option for fluxctl.

**Expected behavior**
Documentation should say how to uninstall Flux, or - if it's impossible - should clearly say that it's not possible. From user point of view the most logical solution would be to add "uninstall" option for fluxctl. But even manual kubectl commands set will be enough

- Fluxctl version: 1.20.0

**Comment by stefanprodan** [comment]
Instead of `apply` do `delete` e.g. `fluxctl install .... | kubectl delete -f -`.

**Comment by Marx2** [comment]
Your advice works ok, thank you. I just suggest to add it also to documentation

**Comment by kingdonb** [comment]
Thanks for your report. Flux v1 is in maintenance mode.

I've recently joined the team to focus on Flux v1 issues, of course we hope you are able to upgrade, here's more info on how to find support with that: https://fluxcd.io/support/

There are many issues that are better addressed in the later releases of Flux, under `fluxcd/flux2` and this is another of them. Now `flux` cli bundles an uninstall command, that features prominently in the documentation and helps take care of cleaning up any CRDs and cluster resources that would be left behind from your installation of Flux.

Welcome to suggest any changes to the documentation, I will review the docs for installing and uninstalling in Flux v1 and see if there is somewhere this change belongs. Closing for now 😃
